go.sum

```diff
@@ -102,10 +102,12 @@ github.com/google/btree v0.0.0-20180124185431-e89373fe6b4a h1:ZJu5NB1Bk5ms4vw0Xu
 github.com/google/btree v0.0.0-20180124185431-e89373fe6b4a/go.mod h1:lNA+9X1NB3Zf8V7Ke586lFgjr2dZNuvo3lPJSGZ5JPQ=
 github.com/google/go-cmp v0.2.0 h1:+dTQ8DZQJz0Mb/HjFlkptS1FeQ4cWSnN941F8aEG4SQ=
 github.com/google/go-cmp v0.2.0/go.mod h1:oXzfMopK8JAjlY9xF4vHSVASa0yLyX7SntLO5aqRK0M=
-github.com/google/go-github v0.0.0-20170604030111-7a51fb928f52 h1:RO1+F2JB1BPRWnm4WDgfkW7Y4l1F+AyemYnlYtmDP3s=
-github.com/google/go-github v0.0.0-20170604030111-7a51fb928f52/go.mod h1:zLgOLi98H3fifZn+44m+umXrS52loVEgC2AApnigrVQ=
-github.com/google/go-querystring v0.0.0-20150414214848-547ef5ac9797 h1:XScp/7K55BJWgFeKalWP/2XvaRdhhhNJBOSQuFMS8mE=
-github.com/google/go-querystring v0.0.0-20150414214848-547ef5ac9797/go.mod h1:odCYkC5MyYFN7vkCjXpyrEuKhc/BUO6wN/zVPAxq5ck=
+github.com/google/go-github v17.0.0+incompatible h1:N0LgJ1j65A7kfXrZnUDaYCs/Sf4rEjNlfyDHW9dolSY=
```

Comment:
I'm not sure why this isn't going away.

Reply:
Maybe something else still still depends on v17?

Reply:
I couldn't find anything...

Reply:
Looks like go-github v24 itself depends on v17: https://github.com/google/go-github/blob/00add49aab2728f7c246b27be612b002b22e7f94/go.mod#L5

Reply:
Wat 🤯

Reply:
lol, what...